service/history/replication/eventhandler/remote_history_paginated_fetcher.go

```diff
@@ -202,7 +202,7 @@ func (n *HistoryPaginatedFetcherImpl) getHistory(
 		return nil, err
 	}
 
-	response, err := adminClient.GetWorkflowExecutionRawHistoryV2(ctx, &adminservice.GetWorkflowExecutionRawHistoryV2Request{
+	response, err := adminClient.GetWorkflowExecutionRawHistory(ctx, &adminservice.GetWorkflowExecutionRawHistoryRequest{
```

Comment:
Is it possible to be gated by a feature flag?

Reply:
yeah. This is not used anywhere yet (except in integration test). I will gate the entire feature where this component are used.